packages/p2p/src/stores/my-profile-store.js

```diff
@@ -353,6 +370,26 @@ export default class MyProfileStore extends BaseStore {
         this.setShouldShowAddPaymentMethodForm(false);
     }
 
+    /**
+     * This function loads more blocked advertisers as necessary if the user is searching for a blocked advertiser
+     * It updates the search_results based on the searched advertiser
+     */
+    @action.bound
+    loadMoreBlockedAdvertisers() {
+        if (this.search_term) {
+            const search_results = this.blocked_advertisers_list.filter(blocked_advertiser =>
+                blocked_advertiser.name.toLowerCase().includes(this.search_term.toLowerCase().trim())
+            );
+
+            if (search_results.length) {
+                this.setSearchResults(search_results);
+            } else {
+                this.setSearchResults([]);
+            }
```

Comment:
Wouldn't `search_results` return an empty array if there is no result? I think we only need `this.setSearchResults(search_results);` here.

Reply:
Oh yaa it gets checked already in `rendered_blocked_advertisers_list` 😅